datasetinsights/commands/download.py

```diff
@@ -4,10 +4,43 @@
 import click
 
 import datasetinsights.constants as const
+from datasetinsights.datasets.base import DownloaderRegistry
+from datasetinsights.datasets.unity_simulation import UnitySimulationDownloader
 
 logger = logging.getLogger(__name__)
 
 
+# class Mutex(click.Option):
```

Comment:
So we don't use this class anymore?

Reply:
I have commentated out for now in this draft PR. The final PR will either have it or not, but its currently unclear as the design of the CLI is not finished.